src/common/c_jhash.h

```diff
@@ -125,7 +125,7 @@
  *            avalanche. About 36+6len instructions.
  */
 static inline uint32_t c_jhash(const uint8_t *k, uint32_t length, uint32_t initval) {
-   uint32_t a,b,c,len;
+   uint32_t a = 0,b = 0,c = 0,len = 0;
```

Comment:
Not your fault since it was already here before your change but since we're here I'll use the opportunity to nitpick a bit. Could we at least have a space after each comma?

Or even better, I think it's best to separate the variable declarations, so what about:
```
uint32_t a = 0;
uint32_t b = 0;
uint32_t c = 0;
uint32_t len = 0;
```
I generally find that more readable.